### PR TITLE
Add ACPT docs to ORT docs

### PR DESCRIPTION
**Pull request by sophies927**
### Description
Add an ACPT landing page to the ORT docs to publish supported configurations + most recent releases

### CODE REVIEW

File: docs/ecosystem/acpt.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Why should I use ACPT?
+* **Flexibility:** Use as-is with preinstalled packages or build on top of the curated environment.
+* **Ease of use:** All components are installed and validated against dozens of Microsoft workloads to reduce setup costs and accelerate time to value.
+* **Efficiency:** Avoid unnecessary image builds and only have required dependencies that are accessible right in the image/container.
+* **Optimized training framework:** Set up, develop, and accelerate PyTorch models on large workloads, and improve training and deployment success rate.
+* **Up-to-date stack:** Access the latest compatible versions of Ubuntu, Python, PyTorch, CUDA/RocM, etc.
+* **Latest training optimization technologies:** Make use of ONNX Runtime, DeepSpeed, MSCCL, and more.
+
+## Supported configurations for Azure Container for PyTorch (ACPT)
+The following configurations are supported in the Microsoft Container Registry (MCR): [ptca_image_list.md](https://aiinfra.visualstudio.com/PyTorch/_git/ptebic?path=/.azure_pipelines/pipeline_config/ptca_image_list.md&_a=preview).
```
Comment (sheetalarkadam) [inline docs/ecosystem/acpt.md:28]: Not sure if this link will be accessible to public

File: docs/ecosystem/acpt.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Most recent image releases (last updated 4/25/2024):
+
+| Environment Name            | OS             | GPU Version | Python Version | PyTorch Version | ORT-training Version | DeepSpeed Version | torch-ort Version | Nebula Version |
+|-----------------------------|----------------|-------------|----------------|-----------------|---------------------|-------------------|-------------------|----------------|
```
Comment (ajindal1) [inline docs/ecosystem/acpt.md:33]: We have updated the ORT-Training version to 1.18.0 and Deepspeed version to 0.14.2. So, we will need to update it here as well as AML page, thanks.

File: docs/ecosystem/acpt.md (hunk 1 above)
Comment (ajindal1) [reply docs/ecosystem/acpt.md:28]: It is not, can't access it through incognito mode.

Comment (sophies927) [reply docs/ecosystem/acpt.md:28]: I thought that might be the case. Should I just add all of the supported configurations listed in that table?

Also, how can people access them in MCR? I tried the links in the table, but it doesn't look like they're accessible to me.

Comment (ajindal1) [reply docs/ecosystem/acpt.md:28]: The links in the table don't work, they don't point to anything.

Comment (sheetalarkadam) [reply docs/ecosystem/acpt.md:28]: The links in the table are not accessible but the mcr docker pull commands will work for all. We should add the corresponding MCR images names here if we want to remove the AML curated environment dependency.

Comment (sheetalarkadam) [reply docs/ecosystem/acpt.md:28]: You can add the file you have linked in this PR. We can add a process to copy this file https://aiinfra.visualstudio.com/PyTorch/_git/ptebic?path=/.azure_pipelines/pipeline_config/ptca_image_list.md to the git repo on a weekly basis

Comment (hanbitmyths) [inline docs/ecosystem/acpt.md:28]: Remove this link since it's an internal link and not accessible from public.

**Review by MaanavD — DISMISSED** [review]
Looked over code, will review some curly braces to make sure they came out right once pushed to prod, but LGTM! 🚀

**Review by sheetalarkadam — APPROVED** [review]

**Comment by sheetalarkadam** [comment]
Looks good, approved!

**Review by MaanavD — APPROVED** [review]
LGTM again!

**Pull request merged**